perun-core/src/main/java/cz/metacentrum/perun/core/impl/modules/attributes/urn_perun_member_attribute_def_def_o365EmailAddresses_mu.java

```diff
@@ -89,25 +102,33 @@ public void checkAttributeSemantics(PerunSessionImpl sess, Member member, Attrib
 		}
 		String ucoEmail = UCO + "@muni.cz";
 		if (!emails.contains(ucoEmail)) {
-			throw new WrongAttributeValueException(attribute, member, "does not contain " + ucoEmail);
+			throw new WrongReferenceAttributeValueException(attribute, "does not contain " + ucoEmail);
```

Comment:
This is really a semantic error, but cause of this error is in the value of attribute attrUCO for member, which is now missing here.

Reply:
Okay, the exception is updated now.